ci-operator/step-registry/cucushift/installer/check/vsphere/folder/cucushift-installer-check-vsphere-folder-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+# shellcheck disable=SC1091
+source "${SHARED_DIR}/vsphere_context.sh"
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+FOLDER="/$vsphere_datacenter/vm/$vsphere_datacenter"
+SUB_FOLDER="$FOLDER/${NAMESPACE}-${UNIQUE_HASH}"
+
+nodes_list=$(oc get nodes -ojson | jq -r '.items[].metadata.name')
+folder_vm_list=$(govc find -type m "$SUB_FOLDER" | awk -F "/" '{print $6}')
```

Comment:
Can we use `govc vm` command to replace `find` command for better performance?

Reply:
I did try to find a better way to get vm names, but do not think `govc vm` works. `# govc ls /DEVQEdatacenter/vm/ci-op-2bt5s8m8-f0130-98cxw` show the same result, is it have a better performance? I'm not sure.

`# govc vm
govc: command 'vm' not found, did you mean:
  cluster.module.vm.add
  cluster.module.vm.rm
  import.vmdk
  library.vmtx.info
  vm.change
  vm.clone
  vm.console
  vm.create
  vm.customize
  vm.destroy
  vm.disk.attach
  vm.disk.change
  vm.disk.create
  vm.guest.tools
  vm.info
  vm.instantclone
  vm.ip
  vm.keystrokes
  vm.markastemplate
  vm.markasvm
  vm.migrate
  vm.network.add
  vm.network.change
  vm.option.info
  vm.power
  vm.question
  vm.rdm.attach
  vm.rdm.ls
  vm.register
  vm.unregister
  vm.upgrade
  vm.vnc`

Reply:
My suggested command does not work, my fault, maybe we can use `govc ls` command to get the list, e.g:
```
[root@preserve-jialiu ~]# govc ls /DEVQEdatacenter/vm/newugd-51163-8bmxz/
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-control-plane-0
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-compute-1
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-control-plane-2
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-compute-2
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-control-plane-1
/DEVQEdatacenter/vm/newugd-51163-8bmxz/newugd-51163-8bmxz-compute-0
/DEVQEdatacenter/vm/newugd-51163-8bmxz/lb-0
```